client/src/components/Endpoints/Icon.tsx

```diff
@@ -33,7 +33,7 @@ const Icon: React.FC<IconProps> = (props) => {
   const avatarSrc = useAvatar(user);
 
   if (isCreatedByUser) {
-    const username = user?.name || 'User';
+    const username = user?.name || user?.username || 'User';
```

Comment:
thanks for this. PR lgtm but noticed this is missing the localization of 'User'

do you mind adding it in by import useLocalize hook directly (not from index)?

```ts
import useLocalize from '~/hooks/useLocalize';
/* ... */
      const localize = useLocalize();
      const username = user?.name || user?.username || localize('com_nav_user');
```

Reply:
I've added it